### PR TITLE
Added new member - Hauwa Ismail Ahmad

### LINKED ISSUE #5071: [Community] Member Profile: Hauwa Ismail Ahmad
**Current Behavior**

Hauwa has been a consistent contributor and community member.

**Desired Situation**
Let's recognize Hauwa as a contributor and community member by creating a profile on https://layer5.io/community/members.

- GitHub: 
- Twitter: [hauwy_ismyl](https://twitter.com/hauwy_ismyl)
- LinkedIn: [hauwa-ismail-75224a255](https://www.linkedin.com/in/hauwa-ismail-75224a255)
- Layer5 Cloud: 
- Link to profile picture:

A detailed explanation on how to set up a community member profile can be found in the [CONTRIBUTING.md](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)

---

**Contributor Resources**

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)
- See Layer5 site designs in this [Figma project](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs). Join the [Layer5 Community](https://slack.layer5.io) for access.

### PR DESCRIPTION
**Pull request by IrfanSalim**
**Description**

This PR fixes #5071 

**Notes for Reviewers**
Added the folder using the profile template, however on the member's page, was not able to see the newly added member.

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit d8201a2c70fbfc15246bb14395103bd3e41de6ed at: https://6535546fd9bdce08bd8c934d--layer5.netlify.app

File: src/collections/members/hauwa-ismail-ahmad/index.mdx [hunk 1]
```diff
@@ -0,0 +1,14 @@
+---
+name: Hauwa Ismail Ahmad
+position: Contributor
+image_path: ./hauwa-ismail-ahmad.webp
+github:
+twitter: hauwy_ismyl
+linkedin: hauwa-ismail-75224a255
+layer5:
```
Comment (abhijeetgauravm) [inline src/collections/members/hauwa-ismail-ahmad/index.mdx:8]: Here empty github user handle, cloud user id and location is causing the error, you are facing in preview!

File: src/collections/members/hauwa-ismail-ahmad/index.mdx [hunk 2]
```diff
@@ -0,0 +1,14 @@
+---
+name: Hauwa Ismail Ahmad
+position: Contributor
+image_path: ./hauwa-ismail-ahmad.webp
+github:
```
Comment (vishalvivekm) [inline src/collections/members/hauwa-ismail-ahmad/index.mdx:5]:
```suggestion
```

File: src/collections/members/hauwa-ismail-ahmad/index.mdx [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+twitter: hauwy_ismyl
+linkedin: hauwa-ismail-75224a255
+layer5:
+location:
```
Comment (vishalvivekm) [inline src/collections/members/hauwa-ismail-ahmad/index.mdx:9]:
```suggestion
location: Sokoto, Nigeria
```

File: src/collections/members/hauwa-ismail-ahmad/index.mdx [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+bio: "Dedicated to helping Layer5 thrive in the digital landscape. Avid reader, music enthusiast, and an admirer of art (not abstract)."
+badges:
+- community
+status: Active
```
Comment (vishalvivekm) [inline src/collections/members/hauwa-ismail-ahmad/index.mdx:13]:
```suggestion
status: Active
published: true
```

**Review by vishalvivekm — CHANGES_REQUESTED** [review]
@IrfanSalim

File: src/collections/members/hauwa-ismail-ahmad/index.mdx (hunk 1 above)
Comment (vishalvivekm) [inline src/collections/members/hauwa-ismail-ahmad/index.mdx:8]:
```suggestion
```

File: src/collections/members/hauwa-ismail-ahmad/index.mdx [hunk 5]
```diff
@@ -0,0 +1,13 @@
+---
+name: Hauwa Ismail Ahmad
+position: Contributor
+image_path: ./hauwa-ismail-ahmad.webp
+twitter: hauwy_ismyl
+linkedin: hauwa-ismail-75224a255
```
Comment (vishalvivekm) [inline src/collections/members/hauwa-ismail-ahmad/index.mdx:6]:
```suggestion
linkedin: hauwa-ismail-ahmad-75224a255
```

**Comment by l5io** [comment]
🚀 Preview for commit 890c659a7e5e0d32b63fd4879ccb07de3406aa94 at: https://65356ba057ebb91a8d9f6eb2--layer5.netlify.app

**Review by vishalvivekm — APPROVED** [review]

**Comment by saurabh100ni** [comment]
Hey @IrfanSalim  Let's discuss this on website call tomorrow at 5:30 PM IST / 7 AM Central time. Please add this as an agenda item in the [meeting minutes](https://docs.google.com/document/d/1XczAHXVe2FIWPqiF57ospJ43zw5cZQ7ui8mn39v5EvA/edit#heading=h.lohhtewfwima).

**Comment by vishalvivekm** [comment]
Thank you! @IrfanSalim

**Comment by IrfanSalim** [comment]
> Thank you! @IrfanSalim

Glad I could help, I added the agenda for the connect today as suggested by @saurabh100ni, should I remove that since this is merged?

**Comment by abhijeetgauravm** [comment]
> > Thank you! @IrfanSalim
> 
> Glad I could help, I added the agenda for the connect today as suggested by @saurabh100ni, should I remove that since this is merged?

Yeah you can @IrfanSalim !

**Pull request merged**